docs/community/content/team/_index.cn.md

```diff
@@ -30,29 +30,30 @@ chapter = true
 
 ### 提交者
 
-| 名字   | GitHub ID   |
-| :---- | :---------- |
```

Comment:
It's better to just add new committer info, just as the title says. Formatting could be a new PR.

Reply:
All right.